proto/money/rpc/reservation_folio_svc.proto

```diff
@@ -168,6 +168,7 @@ enum ReservationFolioSvcAuthorizationModificationResult {
 	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_SUCCESS = 0;
 	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_SALE_NOT_FOUND = 1;
 	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_UNKNOWN_FAILURE = 2;
+	RESERVATION_FOLIO_SVC_AUTHORIZATION_MODIFICATION_DECLINED = 3;
```

Comment:
I didn't want to remove unknown failure, because we use that in the case of an unhandled exception.